### PR TITLE
Integrate websocket client into rails side

### PR DESCRIPTION
**Pull request by ShiningRay**
And remove the usage of `PoolTransactionEntry`, instead using `CkbTransaction.tx_pending`.

### CODE REVIEW

File: app/models/ckb_transaction.rb [hunk 1]
```diff
@@ -53,6 +57,14 @@ def self.cached_find(query_key)
     end
   end
 
+  def self.clean_pending
+    tx_pending.find_each do |t|
+      if tx_committed.where(tx_hash: t.tx_hash).exists? || tx_rejected.where(tx_hash: t.tx_hash).exists?
```
Comment (zmcNotafraid) [inline app/models/ckb_transaction.rb:62]:
```suggestion
      if where(tx_hash: t.tx_hash).where.not(tx_status: :pending).exists?
```

File: app/models/pool_transaction_entry.rb [hunk 2]
```diff
@@ -15,11 +15,42 @@ def income(address = nil)
   end
 
   def display_outputs(previews: false)
-    self.attributes["display_outputs"]
+    cell_inputs_for_display = cell_inputs.to_a.sort_by(&:id)
```
Comment (zmcNotafraid) [inline app/models/pool_transaction_entry.rb:18]: cell_inputs is from where? I found no association with cell_inputs

File: app/models/ckb_sync/new_node_data_processor.rb [hunk 3]
```diff
@@ -282,13 +282,13 @@ def process_withdraw_dao_events!(local_block, new_dao_depositors, dao_contract)
     def process_interest_dao_events!(local_block, dao_contract)
       addrs_withdraw_info = {}
       claimed_compensation = 0
-      local_block.cell_inputs.nervos_dao_withdrawing.select(:id, :ckb_transaction_id,
```
Comment (zmcNotafraid) [inline app/models/ckb_sync/new_node_data_processor.rb:285]: No use for block_id?

File: app/jobs/import_block_job.rb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-class ProcessBlockJob < ApplicationJob
+class ImportBlockJob < ApplicationJob
```
Comment (zmcNotafraid) [inline app/jobs/import_block_job.rb:1]: When to invoke this job?

**Review by zmcNotafraid — CHANGES_REQUESTED** [review]

File: app/models/pool_transaction_entry.rb (hunk 2 above)
Comment (ShiningRay) [reply app/models/pool_transaction_entry.rb:18]: The `pool_transaction_entry` is deprecated, will not be used in the future.
We shall use `CkbTransaction.tx_pending` instead

File: app/models/ckb_sync/new_node_data_processor.rb (hunk 3 above)
Comment (ShiningRay) [reply app/models/ckb_sync/new_node_data_processor.rb:285]: The `dao`  field in `cell_outputs` is redudent to the `blocks.dao`.
I will remove the `cell_outputs.dao` in the future.
And save the parsed dao headers in the `blocks` table.

File: app/jobs/import_block_job.rb (hunk 4 above)
Comment (ShiningRay) [reply app/jobs/import_block_job.rb:1]: This is a placeholder for future refactoring

**Review by zmcNotafraid — APPROVED** [review]

**Pull request merged**